packages/shorebird_cli/README.md

```diff
@@ -94,6 +94,27 @@ shorebird apps create
 Created new app: my-app-id
 ```
 
+### Delete App
+
+To delete an existing app on Shorebird, use the `shorebird apps delete` command. An app-id can be specified as a CLI option but shorebird will default to the product_id defined in the `shorebird.yaml`
+
+```bash
+# Create an app using default app id
+shorebird apps delete
```

Comment:
I wonder if this is "register" "unreigster".  I guess we do delete all the artifacts we store if you delete your apps?

Reply:
Yeah we can change later but I was envisioning we'd be permanently deleting all your artifacts as well 👍